lib/cosmos/interfaces/tcpip_server_interface.rb

```diff
@@ -151,11 +151,15 @@ def stop_raw_logging
 
     # Supported Options
     # LISTEN_ADDRESS - Ip address of the interface to accept connections on - Default: 0.0.0.0
+    # AUTO_SYSTEM_META - Automatically send SYSTEM META on connect - Default false
```

Comment:
This is needed for the DART database so that it always knows what the current META information is.